sdcm/monitorstack/restore.py

```
@@ -368,18 +373,17 @@ def restore_sct_dashboards(grafana_docker_port, sct_dashboard_file):
+    dashboard_url = f"http://localhost:{grafana_docker_port}" + GRAFANA_DASHBOARD_API_PATH
     with open(sct_dashboard_file, encoding="utf-8") as f:
         dashboard_config = json.load(f)
         if dashboard_config.get("dashboard", {}).get("id"):
             dashboard_config["dashboard"]["id"] = None
 
+    payload = convert_dashboard_payload_to_new_api(dashboard_config)
     try:
-        res = requests.post(
-            dashboard_url, data=json.dumps(dashboard_config), headers={"Content-Type": "application/json"}
-        )
+        res = requests.post(dashboard_url, data=json.dumps(payload), headers={"Content-Type": "application/json"})
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)